### ISSUE TITLE
entity-pathing: mobs can still escape from plots after getting attacked by a player

### ISSUE THREAD

**Issue by ofunny** [IntellectualSites/PlotSquared#2848]
**PlotSquared Feature Suggestion:**
Even with entity-pathing: true, all mobs that will follow and attack you (after the player harms them first) will track and follow you out of any plot. Entity-pathing will not work in this circumstance.

**How should it be implemented?**
With entity-pathing on, mobs should not get out of any plot, even if they are in attack mode and track/follow a player. No matter if the player harmed them before or not.

**Additional context**
You can try it out with dogs, cave spider and any other mob that follows you, after you harm it.

**Checklist**:

- [x] I made sure there are no duplicates of my suggestion [(Use search)](https://github.com/IntellectualSites/PlotSquaredSuggestions/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+)
- [x] I made sure my suggestion does not exist yet [(Check the wiki)](https://github.com/IntellectualSites/PlotSquared/wiki)
- [x] I made sure my suggestion is useful for the majority of PlotSquared users

**Comment by RedstoneFuture** [comment]
moved: https://issues.intellectualsites.com/issue/PS-5

**Comment by ofunny** [comment]
Actually, I'm sorry about that report, I really don't know how but GadgetMenu pets do not get removed by PlotSquared kill road mobs feature anymore – I do not know how they fixed that. Normal mobs get removed. Since we talked about it in Discord and you told me it is a very destructive feature and will destroy all mobs, I did not test it again with the newest version. So only the other problem with (riding players can not do plot commands)  is relevant. So that NPC tag will be a thing for other plugins.
